src/main/java/com/spotinst/sdkjava/SpotinstClient.java

```diff
@@ -13,6 +13,10 @@
 public class SpotinstClient {
 
     //region Methods
+    public static SpotInstOceanGKELaunchSpecClient getSpotinstOceanGKELaunchSpecClientTest(String authToken, String account) {
```

Comment:
Rename to return type to SpotOceanGKELaunchSpecClient. Make changes is class name accordingly.
The function name shouldn't contain 'Test'. Rename to getSpotOceanGKELaunchSpecClient.